core/app/models/spree/payment_method.rb

```diff
@@ -73,5 +76,22 @@ def store_credit?
     def available_for_order?(_order)
       true
     end
+
+    def transaction_limits
+      { minimum: transaction_minimum, maximum: transaction_maximum }
+    end
+
+    # Checks if the total of an Order is within the defined Minimum and Maximum Transaction Amount 
+    def within_transaction_limits?(_order)
+      min_valid = transaction_minimum.to_f <= _order.total
```

Comment:
@krtschmr if a variable is used it should not be prepended with an underscore, could you change it to `order` in this method? :)